sdk/src/transaction.rs

```diff
@@ -113,9 +113,15 @@ impl From<SanitizeError> for TransactionError {
 #[frozen_abi(digest = "AAeVxvWiiotwxDLxKLxsfgkA6ndW74nVbaAEb6cwJYqR")]
 #[derive(Debug, PartialEq, Default, Eq, Clone, Serialize, Deserialize, AbiExample)]
 pub struct Transaction {
-    /// A set of digital signatures of `account_keys`, `program_ids`, `recent_blockhash`, and `instructions`, signed by the first
-    /// signatures.len() keys of account_keys
-    /// NOTE: Serialization-related changes must be paired with the direct read at sigverify.
+    /// A set of digital signatures of the [`account_keys`], [`recent_blockhash`], and [`instructions`] fields
+    /// of [`message`], signed by the first `signatures.len()` keys of `account_keys`.
+    ///
+    /// [`account_keys`]: Message::account_keys
+    /// [`recent_blockhash`]: Message::recent_blockhash
+    /// [`instructions`]: Message::instructions
+    /// [`message`]: Transaction::message
```

Comment:
I know the original docs implied otherwise, but the signatures actually include the complete Message, not just the specified fields. How about simply...

```suggestion
    /// A set of digital signatures of a serialized [`Message`],
    /// signed by the first `signatures.len()` keys of `account_keys`.
    ///
    /// [`account_keys`]: Message::account_keys
```